tutorials/data/flow_benchmark_2d_geiger_setup.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
```

Comment:
Moved file from test to avoid having to change python paths to run the tutorials